src/components/signup/SignupSvg.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import commonStyle from '../common/common.scss';
+
+export default function SignupSvg() {
+  return (
+    <div className="msg-svg" style={{ commonStyle }}>
+      <div className="msg-svg__message">
+        <p>If you already have an account!</p>
+      </div>
+      <div className="msg-svg__Login-svg">
+        <a href="/login">
+          <svg
```

Comment:
It would be good if this svg is put in its own file

Reply:
I worked on the feedback